x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/TransactionTabs.tsx

```diff
@@ -74,6 +76,7 @@ const timelineTab = {
 };
 
 const metadataTab = {
+  dataTestSubj: 'transactionDetailsMetadataTab',
```

Comment:
Thanks for adding `dataTestSubj`attributes. We should not hesitate to do this when adding tests.